tsconfig.json

```
@@ -9,6 +9,9 @@
 		"sourceMap": true,
 		"strictNullChecks": true,
```

If possible it'd be nice to make this
```suggestion
		"strict": true,
```
as it's all new code so we should be able to conform to fully strict ts